docs_src/multiple_values/options_with_multiple_values/tutorial002.py

```diff
@@ -0,0 +1,25 @@
+from enum import Enum
+from typing import Tuple
+
+import typer
+
+
+class Food(str, Enum):
+    f1 = "Eggs"
+    f2 = "Bacon"
+    f3 = "Cheese"
+
+
+def main(user: Tuple[str, int, bool, Food] = typer.Option((None, None, None, Food.f1))):
```

Comment:
This file and the next (`tutorial002` and `tutorial003`) are important to test, because they encovered a bug. But it feels overkill to actually mention these in the documentation? So maybe we should move these tests to somewhere else?